cqrs/core/event-store-adapters/MemoryAdapter.js

```diff
@@ -0,0 +1,63 @@
+import {AbstractEventStoreAdapter} from '../abstraction/AbstractEventStoreAdapter';
+
+class MemoryAdapter extends AbstractEventStoreAdapter {
+
+    constructor (eventBus) {
+        super(eventBus);
+        this._aggregates = {};
+        this._events = [];
+        this._eventBus = eventBus;
+    }
+
+    readAllEvents (uuid) {
+        if (!this.isAggregateRegistered(uuid)) return null;
+        return this._getAggregateEvents(uuid);
+    }
+
+    writeEvents (uuid, originalVersion = 0, events) {
+        return new Promise ((resolve, reject) => {
+            if (!this.isAggregateRegistered(uuid)) this._registerNewAggregate(uuid);
```

Comment:
It will be always true, because isAggregateRegistered method returns Promise objects, and objects are always true.